frontend/src/components/PenRequestDetail.vue

```diff
@@ -817,6 +824,22 @@ export default {
           this.loadingActionResults = false;
         });
     },
+    searchDuplicatePenRequestsByPen() {
+      this.loadingDuplicatePenRequests = true;
+      ApiService.apiAxios
+        .get(`${Routes.PEN_REQUEST_SEARCH_URL}duplicatePenRequests?pen=${this.penSearchId}`)
```

Comment:
The axios get function accepts params as an argument, so they don't have to be in the url string